File: starterkits/csharp/game/Colony.cs
using System.Collections.Generic;
using static Blitz2021.Map;

namespace Blitz2021
{
    public class Crew
    {
        public class Prices
        {
            public int MINER;
            public int OUTLAW;
            public int CART;
        }

        public string id;
        public string name;
        public Position homeBase;
        public int blitzium;
        public int totalBlitzium;
        public List<Unit> units;
        public List<string> errors;
        public Prices prices;
    }

    public class Unit
    {
        public enum UnitType
        {
            MINER, CART, OUTLAW
        }

        public string id;
        public UnitType type;
        public int blitzium;
        public Position position;
        public List<Position> path;
    }
}

File: starterkits/java/src/main/java/codes/blitz/game/message/game/Colony.java
package codes.blitz.game.message.game;

import java.util.List;

public class Colony {
    private String id;
    private String name;
    private Position homeBase;
    private Position spawnPoint;
    private int blitzium;
    private int totalBlitzium;
    private List<Unit> units;
    private List<String> errors;


    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Position getHomeBase() {
        return homeBase;
    }

    public void setHomeBase(Position homeBase) {
        this.homeBase = homeBase;
    }

    public Position getSpawnPoint() {
        return spawnPoint;
    }

    public void setSpawnPoint(Position spawnPoint) {
        this.spawnPoint = spawnPoint;
    }

    public int getBlitzium() {
        return blitzium;
    }

    public void setBlitzium(int blitzium) {
        this.blitzium = blitzium;
    }

    public int getTotalBlitzium() {
        return totalBlitzium;
    }

    public void setTotalBlitzium(int totalBlitzium) {
        this.totalBlitzium = totalBlitzium;
    }

    public List<Unit> getUnits() {
        return units;
    }

    public void setUnits(List<Unit> units) {
        this.units = units;
    }

    public List<String> getErrors() {
        return errors;
    }

    public void setErrors(List<String> errors) {
        this.errors = errors;
    }

}
